### ISSUE TITLE
Does the Metal Slug recipe give too few Metal Slugs?

### ISSUE THREAD

**Issue by Cruuk** [Tslat/Advent-Of-Ascension#1278]
**Describe/name the existing feature this suggestion relates to**
Metal Slug

**Describe the change or feature you think would be good to see**
I've crafted various types of ammo in my survival world. Stuff like Limonite Bullets seems fine, Cannonballs are a bit more costly but it doesn't seem too bad otherwise. Haven't really made much Holly arrows and Spreadshots, but the former seems fine considering how easy it is to get lots of the materials fast. Not sure about Spreadshots, but maybe it's a bit fine considering the amount of Limonite bullets you get per bullet recipe?
And Grenades are pretty fine considering the Unstable Gunpowder recipe lol

However, the one that stuck out to me was Metal Slug. The recipe only gives 4, which has felt quite restrictive compared to the others. Even if it is a cheap recipe on its own, I found my Limonite supply quickly going down when trying to make lots of it, to the point where I started to rely on Weapon boxes. Thanks to my luck, it kept giving me everything else other than Metal Slug (I got lots of cannonballs, maybe my view of how expensive cannonballs are could be skewed because of weapon box?). I guess Assassin does seem like a decent, or maybe even great source of them, but... yeah.

Basically, Metal slug feels a bit too restrictive compared to other ammo. Even with the slow fire rate of Snipers it felt hard trying to make the ammo supply not go down fast. I'd say the recipe should give **6-8** instead of 4. I'm not sure if there's any other issues with the other ammo recipes, but this one is the most glaring to me.

**Comment by Immortallis1** [comment]
I'll second this. Metal Slugs feel slightly less worth it than they should.

**Comment by Tslat** [comment]
I was a bit on edge about this one but I've done it
